apps/calendar/js/views/modify_event.js

```diff
@@ -49,17 +53,20 @@ ModifyEvent.prototype = {
 
   _duration: 0, // The duration between start and end dates.
 
+  removeDialogClass: 'remove-dialog',
+
   _initEvents: function() {
     EventBase.prototype._initEvents.apply(this, arguments);
 
     var calendars = this.app.store('Calendar');
-
     calendars.on('add', this._addCalendarId.bind(this));
     calendars.on('preRemove', this._removeCalendarId.bind(this));
     calendars.on('remove', this._removeCalendarId.bind(this));
     calendars.on('update', this._updateCalendarId.bind(this));
 
     this.deleteButton.addEventListener('click', this.deleteRecord);
+    this.deleteRecordButton.addEventListener('click', this.hideHeaderAndForm);
+    this.cancelDeleteButton.addEventListener('click', this.cancelDelete);
```

Comment:
also when you click "cancel" the view flashes... made me wonder if we shouldn't simply use the `window.confirm()` since that would be way less lines of code..